src/pages/add-wallet/add-wallet.ts

```diff
@@ -10,7 +10,9 @@ import { ViewController } from 'ionic-angular';
 export class AddWalletPage implements OnInit {
 
   form: FormGroup;
-  seed: string;
+  label: string="";
```

Comment:
There's no need to type cast as string when providing an empty string, it will auto type cast. Also add spaces around equals sign (label = "");